runtime/balrt.h

```diff
@@ -281,6 +281,11 @@ typedef struct {
     PanicCode panicCode;
 } TaggedPtrPanicCode;
 
+typedef struct {
+    int64_t val;
```

Comment:
Don't abbreviate `val` to `value` when the other field is unabbreviated and longer than `value`.

Reply:
Changed